### PR TITLE
Feature/terraform runner example

### PR DESCRIPTION
**Pull request by felixzieger**
The new command is referenced in https://github.com/meshcloud/unipipe-service-broker/wiki/Getting-started-with-unipipe-terraform-runner-on-Azure

### CODE REVIEW

File: cli/unipipe/blueprints/terraform-runner-hello-world-service.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      longDescription: "This simple service renders recipes in your git repository. It uses a terraform script that is executed by terraform-runner. Have a look in the terraform/ folder of your git repository to find out more."
+      providerDisplayName: "Pizza Inc."
+      documentationUrl: "https://docs.microsoft.com/en-us/azure/devops/?view=azure-devops"
```
Comment (sttomm) [inline cli/unipipe/blueprints/terraform-runner-hello-world-service.ts:17]: d: Should we skip the URLs here. I guess links to Azure DevOps are more confusing than helpful here?

File: cli/unipipe/blueprints/terraform-runner-hello-world-service.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+}
+
+resource "local_file" "pizza_receipe" {
+  content  = <<EOT
+Preheat the oven to 200 degrees Celsius
+Form the pizza dough and place it on a peel dusted with a little flour. 
+Add sauce, cheese and \${var.topping}.
+Slip the pizza into the oven
+After 10 minutes of baking, check the pizza.
+EOT
```
Comment (sttomm) [inline cli/unipipe/blueprints/terraform-runner-hello-world-service.ts:73]: d: I am wondering whether we could also show usage of e.g. project_id and customer_id here. That could e.g. be the part of "Deliver pizza to \${var.customer_id}.\${var.project_id}"

File: cli/unipipe/blueprints/terraform-runner-hello-world-service.ts [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+variable "plan_name" {
+  description = "Name of the service plan used for the instance."
+  type        = string
+}
+
+#
+# this variable must be set as an environment variable.
+#
+variable "platform_secret" {
+  description = "Client Secret of the Azure App Registration."
```
Comment (sttomm) [inline cli/unipipe/blueprints/terraform-runner-hello-world-service.ts:123]: d: Should be renamed to e.g. "Secret to unlock the Pizza oven" or similar?

File: cli/unipipe/blueprints/terraform-runner-hello-world-service.ts [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+variable "topping" {
+  description = "What goes onto the pizza."
+  type        = string
+}
+
+#
+# the marketplace provides these variables when the service is created
+#
+variable "tenant_id" {
+  description = "Subscription ID of the subscription the network shall be created in."
```
Comment (sttomm) [inline cli/unipipe/blueprints/terraform-runner-hello-world-service.ts:90]: d: For the pizza usacase this should perhpas have a differen description? E.g. "Order ID of the Pizza" or similar?

**Review by sttomm — APPROVED** [review]
lgtm, only some minor improvement ideas

**Comment by felixzieger** [comment]
Thanks for the review comments, @sttomm. I made some changes.

**Pull request merged**